app/models/enterprise_relationship.rb

```diff
@@ -80,6 +81,12 @@ def permissions_list=(perms)
     end
   end
 
+  def revoke_permission(permission)
+    return if permission.nil?
+
+    permissions.where(name: permission).destroy_all
+  end
+
```

Comment:
While I thought that this method is a good addition, it's only used once in a spec. If we don't have a use for it in production code then we shouldn't have this method, I'm realising now.

Reply:
It looks like this method is not needed, so should be removed.